lib/spack/docs/environments.rst

```diff
@@ -72,7 +72,8 @@ Spack then creates the directory ``var/spack/environments/myenv``.
 
 .. note::
 
-   All named environments are stored in the ``var/spack/environments`` folder.
+   All named environments by default are stored in the ``var/spack/environments`` folder.
```

Comment:
Since you're changing this, do we want this to be `managed` (not `named`) environments?

(https://spack-tutorial.readthedocs.io/en/latest/tutorial_environments.html#managed-versus-independent-environments).

Reply:
Done!